website/docs/jellyfish/api/wallet.md

```diff
@@ -29,6 +29,34 @@ interface wallet {
 }
 ```
 
+## getBalances
+
+Returns an object with all balances.
```

Comment:
No full stop

Reply:
But every other method description here has a full-stop.

Reply:
@jingyi2811 you need to communicate why don't put it instead of just saying it.

https://github.com/DeFiCh/whale/pull/58#discussion_r629503905

Reply:
I am okay with whether you add a full stop or not. The thing is you need to consistent with your ts file.